packages/bpk-component-aria-live/src/BpkAriaLive.tsx

```diff
@@ -29,38 +29,32 @@ export const POLITENESS_SETTINGS = {
   off: 'off',
   polite: 'polite',
   assertive: 'assertive',
-};
+} as const;
+
+export type PolitenessSetting =
+  typeof POLITENESS_SETTINGS[keyof typeof POLITENESS_SETTINGS];
 
 export type Props = {
-  children: Node,
-  politenessSetting: $Keys<typeof POLITENESS_SETTINGS>,
-  visible: boolean,
-  className: ?string,
+  children: ReactElement | string;
+  politenessSetting?: PolitenessSetting;
+  visible?: boolean;
+  className?: string | null;
+  [rest: string]: any; // Inexact rest. See decisions/inexact-rest.md
 };
-const BpkAriaLive = (props: Props) => {
-  const { className, politenessSetting, visible, ...rest } = props;
+
+const BpkAriaLive = ({
+  className = null,
+  politenessSetting = POLITENESS_SETTINGS.polite,
+  visible = false,
+  ...rest
+}: Props) => {
   const classNames = getClassName(
     'bpk-aria-live',
     !visible && 'bpk-aria-live--invisible',
     className,
   );
 
-  return (
-    // $FlowFixMe[cannot-spread-inexact] - inexact rest. See 'decisions/flowfixme.md'.
-    <div aria-live={politenessSetting} className={classNames} {...rest} />
-  );
-};
-
-BpkAriaLive.propTypes = {
-  className: PropTypes.string,
-  politenessSetting: PropTypes.oneOf(Object.keys(POLITENESS_SETTINGS)),
-  visible: PropTypes.bool,
-};
-
-BpkAriaLive.defaultProps = {
```

Comment:
Stop using soon-to-be-deprecated .defaultProps and start using the recommended default arguments.

https://github.com/reactjs/rfcs/blob/createlement-rfc/text/0000-create-element-changes.md#deprecate-defaultprops-on-function-components